src/assets/img/index.ts

```diff
@@ -0,0 +1,19 @@
+import IconBackground from './background.jpg'
+import IconElectron from './electron.png'
+import IconIpfs from './ipfs.png'
+import IconOrbit from './orbit.png'
+import IconOrbit2 from './orbit2.png'
+import IconReact from './react.png'
+import IconWatchit from './watchitlogo.png'
+
+const ICONS = {
+    IconBackground, 
+    IconElectron, 
+    IconIpfs, 
+    IconOrbit, 
+    IconOrbit2, 
+    IconReact, 
+    IconWatchit 
+}
+
+export default ICONS
```

Comment:
I like the idea of export all images but are named wrong, icons go somewhere else, here it should name IMAGES and export convention should be like ImgBackground, ImgElectron, ImgIpfs

Reply:
and add empty line at end